docs/DesignDocs/oe_log_message()_callback_proposal.md

```diff
@@ -0,0 +1,61 @@
+Proposal of Public Callback API for oe_log_message()
+====
+
+# Motivation
+
+This document describes the proposed public callback API for oe_log_message().
+
+There are several problems about the current logging that prevent end users to
+make full use of log:
+- It does't have mechanism to communicate between enclave and host.
```

Comment:
The current log calls printf, so that is an ocall from enclave to host. Maybe you can elaborate what you mean over here.

Reply:
Echo @radhikaj. Can you elaborate on `mechanism to communicate between enclave and host`?

Reply:
Updated, thanks!